.github/workflows/release.yml

```diff
@@ -272,7 +272,7 @@ jobs:
       - name: Verify zap-cli exists in Windows arm64 .zip package
         if: startsWith(matrix.os, 'macos')
         run: |
-          output=$(./node_modules/7zip-bin/mac/x64/7za l ./dist/zap-win-arm64.zip)
+          output=$(7za l ./dist/zap-win-arm64.zip)
```

Comment:
I thought that's what `npx` is for? Shouldn't you do `npx 7za` ? I am confused why are you changing this path.

It is true, that running `./node_modules` like it was is wrong, but `npx` should do that for you.

Reply:
https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md

7zip comes pre-installed as a utility in the macos-14 environment. I could change it to use the node module though.